dropgz/build/cni.Dockerfile

```diff
@@ -8,17 +8,21 @@ RUN CGO_ENABLED=0 go build -a -o bin/azure-ipam -trimpath -ldflags "-X main.vers
 
 FROM mcr.microsoft.com/oss/go/microsoft/golang:1.18 AS azure-vnet
 ARG VERSION
+ARG OS
+ARG ARCH
 WORKDIR /azure-container-networking
 COPY . .
-RUN CGO_ENABLED=0 go build -a -o bin/azure-vnet -trimpath -ldflags "-X main.version="$VERSION"" -gcflags="-dwarflocationlists=true" cni/network/plugin/main.go
+RUN curl -LO https://github.com/Azure/azure-container-networking/releases/download/v1.4.29/azure-vnet-cni-swift-$OS-$ARCH-v1.4.29.tgz && tar -xvf azure-vnet-cni-swift-$OS-$ARCH-v1.4.29.tgz
 
 FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS compressor
 WORKDIR /dropgz
 COPY dropgz .
 COPY --from=azure-ipam /azure-ipam/*.conflist pkg/embed/fs
 COPY --from=azure-ipam /azure-ipam/bin/* pkg/embed/fs
 COPY --from=azure-vnet /azure-container-networking/cni/*.conflist pkg/embed/fs
```

Comment:
it would be nice to parameterize the conflist copy by $OS and copy to a generic name (eg `azure-linux-swift.conflist` -> `azure-swift.conflist`) to make the installation that little bit more platform-agnostic